packages/itwinui-react/src/core/LabeledSelect/LabeledSelect.tsx

```diff
@@ -108,29 +111,36 @@ export const LabeledSelect = <T,>(
   };
 
   return (
-    <InputContainer
-      label={label}
-      disabled={disabled}
-      required={required}
-      status={status}
-      message={message}
-      icon={displayStyle === 'default' ? icon() : undefined}
-      isLabelInline={displayStyle === 'inline'}
+    <InputGrid
+      labelPlacement={displayStyle}
       className={className}
       style={style}
-      labelId={labelId}
     >
+      {label && (
+        <Label as='div' required={required} disabled={disabled} id={labelId}>
```

Comment:
what is the significance of making this a div?

Reply:
This comment related: https://github.com/iTwin/iTwinUI/pull/1355#discussion_r1253534607